scripts/config/facet-deploy.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Config file used to upgrade the facets
+ *
+ * - noArgFacets: list of facet names that don't expect any argument passed into initializer
+ * - argFacets: object that specify facet names and arguments that needs to be passed into initializer in format object {facetName: initializerArguments}
+ */
+module.exports = {
+  noArgFacets: [
+    "AccountHandlerFacet",
+    "SellerHandlerFacet",
+    "BuyerHandlerFacet",
+    "DisputeResolverHandlerFacet",
+    "AgentHandlerFacet",
+    "BundleHandlerFacet",
+    "DisputeHandlerFacet",
+    "ExchangeHandlerFacet",
+    "FundsHandlerFacet",
+    "GroupHandlerFacet",
+    "OfferHandlerFacet",
+    "OrchestrationHandlerFacet",
+    "TwinHandlerFacet",
+    "PauseHandlerFacet",
+    "MetaTransactionsHandlerFacet",
+  ],
+  argFacets: {},
```

Comment:
I think it's a good idea to have an example config in the comments.

Reply:
Done.